pageserver/src/walingest.rs

```diff
@@ -53,6 +57,9 @@ pub struct WalIngest {
     shard: ShardIdentity,
     checkpoint: CheckPoint,
     checkpoint_modified: bool,
+    warn_lag_ratelimit: RateLimit,
```

Comment:
Might be overkill to have three separate rate limits rather than just one general one for warnings about timestamps?  Don't feel super strongly.